aikau/src/test/resources/testApp/js/aikau/testing/services/ThrowsInitException.js

```diff
@@ -0,0 +1,41 @@
+/*globals Alfresco*/
+/**
+ * Copyright (C) 2005-2016 Alfresco Software Limited.
+ *
+ * This file is part of Alfresco
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @module aikauTesting/services/ThrowsInitException
+ * @extends module:alfresco/services/BaseService
+ * @author Dave Draper
+ */
+define(["dojo/_base/declare",
+        "alfresco/services/BaseService"],
+        function(declare, BaseService) {
+   
+   return declare([BaseService], {
+      
+      /**
+       * Intentionally causes an exception for testing purposes
+       * 
+       * @instance
+       */
+      initService: function aikauTesting_services_ThrowsInitException__initService() {
+         this.willSurelyDoIt();
```

Comment:
How about `throw new Error("This is an error");`? The current code looks a tad arbitrary (like maybe it's a function defined elsewhere??